packages/steveo/src/consumers/kafka.ts

```diff
@@ -186,6 +186,12 @@ class KafkaRunner
       return;
     }
 
+    if (this.state === 'paused') {
+      this.logger.debug('Consumer paused');
+      this.consumer.consume(1, this.consumeCallback);
```

Comment:
Do we need a sleep in here? Afaik there is no rate limiting on this otherwise so a "paused" consumer is going to just slam kafka for the same message as fast as it can until it is resumed

Reply:
This also suggests that the manager class/behaviour will need to be changed to properly support push-model consumers like kafka as they don't have the polling loop. Ideally we'd be able to support actual kafka pause/resume

Reply:
Sleep makes sense to me. There isn't a consumer poll interval as you said, we are not utilising the stream flowing method so it isn't intelligent enough. 
Agree on manager actually utilising the library methods.